tests/std/tests/P0429R9_flat_map/test.cpp

```diff
@@ -431,13 +439,21 @@ void test_insert() {
     assert(it2->first == 10);
     assert(it2->second == 'n');
 
+    flat_multimap<int, char>::iterator it3 = fmm.insert(fmm.cend(), {70, 'p'});
+    assert(it3->first == 70);
+    assert(it3->second == 'p');
+
     const flat_multimap<int, char>::value_type val_pair2{90, 'w'};
-    const auto it3 = fmm.insert(val_pair2);
-    assert(it3->first == 90);
-    assert(it3->second == 'w');
+    const auto it4 = fmm.insert(val_pair2);
+    assert(it4->first == 90);
+    assert(it4->second == 'w');
+
+    flat_multimap<int, char>::iterator it5 = fmm.insert(fmm.cend(), std::pair<char, char>{static_cast<char>(70), 'q'});
+    assert(it5->first == 70);
+    assert(it5->second == 'q');
 
-    assert(check_key_content(fmm, {10, 10, 90}));
-    assert(check_value_content(fmm, {'n', 'm', 'w'}));
+    assert(check_key_content(fmm, {10, 10, 70, 70, 90}));
+    assert(check_value_content(fmm, {'n', 'm', 'q', 'p', 'w'}));

```

Comment:
Is the particular order of q and p guaranteed by the standard? If not, should use the other overload of check_value_content.

Reply:
I think not, but I see that the order of 'n' and 'm' was checked here. I will check the standard.

Reply:
Yes, I think the intention of "as close as possible" is exactly that

Reply:
You may add `// FIXME` comment here, if you don't want to fix it right away in this PR

Reply:
Added.